/opt/slurm/var/spool/job38575780/slurm_script: line 11: module: command not found
Python 3.10.14
Mon Jul 29 13:41:23 2024       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 470.141.03   Driver Version: 470.141.03   CUDA Version: 11.4     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  Tesla V100-SXM2...  On   | 00000000:06:00.0 Off |                    0 |
| N/A   40C    P0    42W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   1  Tesla V100-SXM2...  On   | 00000000:07:00.0 Off |                    0 |
| N/A   43C    P0    43W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   2  Tesla V100-SXM2...  On   | 00000000:0A:00.0 Off |                    0 |
| N/A   43C    P0    45W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   3  Tesla V100-SXM2...  On   | 00000000:0B:00.0 Off |                    0 |
| N/A   40C    P0    46W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   4  Tesla V100-SXM2...  On   | 00000000:85:00.0 Off |                    0 |
| N/A   43C    P0    44W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   5  Tesla V100-SXM2...  On   | 00000000:86:00.0 Off |                    0 |
| N/A   44C    P0    44W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   6  Tesla V100-SXM2...  On   | 00000000:89:00.0 Off |                    0 |
| N/A   46C    P0    47W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   7  Tesla V100-SXM2...  On   | 00000000:8A:00.0 Off |                    0 |
| N/A   41C    P0    46W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
Traceback (most recent call last):
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/train.py", line 14, in <module>
    from visnet import datasets, models, priors
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/__init__.py", line 1, in <module>
    from .chignolin import Chignolin
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/chignolin.py", line 4, in <module>
    from torch_geometric.data import Data, InMemoryDataset
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/__init__.py", line 6, in <module>
    import torch_geometric.transforms
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/__init__.py", line 38, in <module>
    from .delaunay import Delaunay
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/delaunay.py", line 1, in <module>
    import scipy.spatial
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/__init__.py", line 110, in <module>
    from ._kdtree import *
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_kdtree.py", line 4, in <module>
    from ._ckdtree import cKDTree, cKDTreeNode
ImportError: /usr/local/gcc/5.4.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_ckdtree.cpython-310-x86_64-linux-gnu.so)
Traceback (most recent call last):
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/train.py", line 14, in <module>
    from visnet import datasets, models, priors
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/__init__.py", line 1, in <module>
    from .chignolin import Chignolin
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/chignolin.py", line 4, in <module>
    from torch_geometric.data import Data, InMemoryDataset
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/__init__.py", line 6, in <module>
    import torch_geometric.transforms
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/__init__.py", line 38, in <module>
    from .delaunay import Delaunay
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/delaunay.py", line 1, in <module>
    import scipy.spatial
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/__init__.py", line 110, in <module>
    from ._kdtree import *
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_kdtree.py", line 4, in <module>
    from ._ckdtree import cKDTree, cKDTreeNode
ImportError: /usr/local/gcc/5.4.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_ckdtree.cpython-310-x86_64-linux-gnu.so)
Traceback (most recent call last):
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/train.py", line 14, in <module>
    from visnet import datasets, models, priors
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/__init__.py", line 1, in <module>
    from .chignolin import Chignolin
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/chignolin.py", line 4, in <module>
    from torch_geometric.data import Data, InMemoryDataset
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/__init__.py", line 6, in <module>
    import torch_geometric.transforms
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/__init__.py", line 38, in <module>
    from .delaunay import Delaunay
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/delaunay.py", line 1, in <module>
    import scipy.spatial
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/__init__.py", line 110, in <module>
    from ._kdtree import *
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_kdtree.py", line 4, in <module>
    from ._ckdtree import cKDTree, cKDTreeNode
ImportError: /usr/local/gcc/5.4.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_ckdtree.cpython-310-x86_64-linux-gnu.so)
Traceback (most recent call last):
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/train.py", line 14, in <module>
    from visnet import datasets, models, priors
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/__init__.py", line 1, in <module>
    from .chignolin import Chignolin
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/chignolin.py", line 4, in <module>
    from torch_geometric.data import Data, InMemoryDataset
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/__init__.py", line 6, in <module>
    import torch_geometric.transforms
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/__init__.py", line 38, in <module>
    from .delaunay import Delaunay
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/delaunay.py", line 1, in <module>
    import scipy.spatial
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/__init__.py", line 110, in <module>
    from ._kdtree import *
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_kdtree.py", line 4, in <module>
    from ._ckdtree import cKDTree, cKDTreeNode
ImportError: /usr/local/gcc/5.4.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_ckdtree.cpython-310-x86_64-linux-gnu.so)
Traceback (most recent call last):
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/train.py", line 14, in <module>
    from visnet import datasets, models, priors
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/__init__.py", line 1, in <module>
    from .chignolin import Chignolin
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/chignolin.py", line 4, in <module>
    from torch_geometric.data import Data, InMemoryDataset
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/__init__.py", line 6, in <module>
    import torch_geometric.transforms
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/__init__.py", line 38, in <module>
    from .delaunay import Delaunay
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/delaunay.py", line 1, in <module>
    import scipy.spatial
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/__init__.py", line 110, in <module>
    from ._kdtree import *
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_kdtree.py", line 4, in <module>
    from ._ckdtree import cKDTree, cKDTreeNode
ImportError: /usr/local/gcc/5.4.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_ckdtree.cpython-310-x86_64-linux-gnu.so)
Traceback (most recent call last):
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/train.py", line 14, in <module>
    from visnet import datasets, models, priors
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/__init__.py", line 1, in <module>
    from .chignolin import Chignolin
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/chignolin.py", line 4, in <module>
    from torch_geometric.data import Data, InMemoryDataset
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/__init__.py", line 6, in <module>
    import torch_geometric.transforms
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/__init__.py", line 38, in <module>
    from .delaunay import Delaunay
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/delaunay.py", line 1, in <module>
    import scipy.spatial
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/__init__.py", line 110, in <module>
    from ._kdtree import *
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_kdtree.py", line 4, in <module>
    from ._ckdtree import cKDTree, cKDTreeNode
ImportError: /usr/local/gcc/5.4.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_ckdtree.cpython-310-x86_64-linux-gnu.so)
Traceback (most recent call last):
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/train.py", line 14, in <module>
    from visnet import datasets, models, priors
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/__init__.py", line 1, in <module>
    from .chignolin import Chignolin
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/chignolin.py", line 4, in <module>
    from torch_geometric.data import Data, InMemoryDataset
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/__init__.py", line 6, in <module>
    import torch_geometric.transforms
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/__init__.py", line 38, in <module>
    from .delaunay import Delaunay
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/delaunay.py", line 1, in <module>
    import scipy.spatial
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/__init__.py", line 110, in <module>
    from ._kdtree import *
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_kdtree.py", line 4, in <module>
    from ._ckdtree import cKDTree, cKDTreeNode
ImportError: /usr/local/gcc/5.4.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_ckdtree.cpython-310-x86_64-linux-gnu.so)
Traceback (most recent call last):
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/train.py", line 14, in <module>
    from visnet import datasets, models, priors
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/__init__.py", line 1, in <module>
    from .chignolin import Chignolin
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/chignolin.py", line 4, in <module>
    from torch_geometric.data import Data, InMemoryDataset
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/__init__.py", line 6, in <module>
    import torch_geometric.transforms
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/__init__.py", line 38, in <module>
    from .delaunay import Delaunay
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/delaunay.py", line 1, in <module>
    import scipy.spatial
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/__init__.py", line 110, in <module>
    from ._kdtree import *
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_kdtree.py", line 4, in <module>
    from ._ckdtree import cKDTree, cKDTreeNode
ImportError: /usr/local/gcc/5.4.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_ckdtree.cpython-310-x86_64-linux-gnu.so)
Traceback (most recent call last):
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/train.py", line 14, in <module>
    from visnet import datasets, models, priors
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/__init__.py", line 1, in <module>
    from .chignolin import Chignolin
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/chignolin.py", line 4, in <module>
    from torch_geometric.data import Data, InMemoryDataset
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/__init__.py", line 6, in <module>
    import torch_geometric.transforms
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/__init__.py", line 38, in <module>
    from .delaunay import Delaunay
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/delaunay.py", line 1, in <module>
    import scipy.spatial
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/__init__.py", line 110, in <module>
    from ._kdtree import *
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_kdtree.py", line 4, in <module>
    from ._ckdtree import cKDTree, cKDTreeNode
ImportError: /usr/local/gcc/5.4.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_ckdtree.cpython-310-x86_64-linux-gnu.so)
Traceback (most recent call last):
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/train.py", line 14, in <module>
    from visnet import datasets, models, priors
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/__init__.py", line 1, in <module>
    from .chignolin import Chignolin
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/chignolin.py", line 4, in <module>
    from torch_geometric.data import Data, InMemoryDataset
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/__init__.py", line 6, in <module>
    import torch_geometric.transforms
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/__init__.py", line 38, in <module>
    from .delaunay import Delaunay
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/delaunay.py", line 1, in <module>
    import scipy.spatial
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/__init__.py", line 110, in <module>
    from ._kdtree import *
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_kdtree.py", line 4, in <module>
    from ._ckdtree import cKDTree, cKDTreeNode
ImportError: /usr/local/gcc/5.4.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_ckdtree.cpython-310-x86_64-linux-gnu.so)
Traceback (most recent call last):
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/train.py", line 14, in <module>
    from visnet import datasets, models, priors
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/__init__.py", line 1, in <module>
    from .chignolin import Chignolin
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/chignolin.py", line 4, in <module>
    from torch_geometric.data import Data, InMemoryDataset
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/__init__.py", line 6, in <module>
    import torch_geometric.transforms
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/__init__.py", line 38, in <module>
    from .delaunay import Delaunay
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/delaunay.py", line 1, in <module>
    import scipy.spatial
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/__init__.py", line 110, in <module>
    from ._kdtree import *
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_kdtree.py", line 4, in <module>
    from ._ckdtree import cKDTree, cKDTreeNode
ImportError: /usr/local/gcc/5.4.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_ckdtree.cpython-310-x86_64-linux-gnu.so)
Traceback (most recent call last):
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/train.py", line 14, in <module>
    from visnet import datasets, models, priors
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/__init__.py", line 1, in <module>
    from .chignolin import Chignolin
  File "/fs03/mo99/tsee0001/Visnet/AI2BMD-ViSNet/visnet/datasets/chignolin.py", line 4, in <module>
    from torch_geometric.data import Data, InMemoryDataset
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/__init__.py", line 6, in <module>
    import torch_geometric.transforms
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/__init__.py", line 38, in <module>
    from .delaunay import Delaunay
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/torch_geometric/transforms/delaunay.py", line 1, in <module>
    import scipy.spatial
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/__init__.py", line 110, in <module>
    from ._kdtree import *
  File "/scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_kdtree.py", line 4, in <module>
    from ._ckdtree import cKDTree, cKDTreeNode
ImportError: /usr/local/gcc/5.4.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /scratch/mo99/tsee0001/miniconda/conda/envs/visnet_go_0/lib/python3.10/site-packages/scipy/spatial/_ckdtree.cpython-310-x86_64-linux-gnu.so)
Mon Jul 29 14:44:23 2024       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 470.141.03   Driver Version: 470.141.03   CUDA Version: 11.4     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  Tesla V100-SXM2...  On   | 00000000:06:00.0 Off |                    0 |
| N/A   36C    P0    41W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   1  Tesla V100-SXM2...  On   | 00000000:07:00.0 Off |                    0 |
| N/A   38C    P0    42W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   2  Tesla V100-SXM2...  On   | 00000000:0A:00.0 Off |                    0 |
| N/A   38C    P0    44W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   3  Tesla V100-SXM2...  On   | 00000000:0B:00.0 Off |                    0 |
| N/A   36C    P0    45W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   4  Tesla V100-SXM2...  On   | 00000000:85:00.0 Off |                    0 |
| N/A   38C    P0    43W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   5  Tesla V100-SXM2...  On   | 00000000:86:00.0 Off |                    0 |
| N/A   39C    P0    43W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   6  Tesla V100-SXM2...  On   | 00000000:89:00.0 Off |                    0 |
| N/A   40C    P0    45W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   7  Tesla V100-SXM2...  On   | 00000000:8A:00.0 Off |                    0 |
| N/A   38C    P0    45W / 300W |      0MiB / 16160MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
slurmstepd: error: *** JOB 38575780 ON dgx000 CANCELLED AT 2024-07-29T15:36:05 ***
